src/v/cluster/health_monitor_frontend.cc

```diff
@@ -147,4 +147,25 @@ ss::future<bool> health_monitor_frontend::does_raft0_have_leader() {
       [](health_monitor_backend& be) { return be.does_raft0_have_leader(); });
 }
 
+ss::future<> health_monitor_frontend::refresh_info() {
```

Comment:
i don't see any functional difference between this and `update_disk_health_cache`. am i missing something? should they be unified?

Reply:
I thought the same thing, but this change is in response to this comment
https://github.com/redpanda-data/redpanda/pull/9480#discussion_r1139972234

Reply:
Resolved offline with Michal and Noah. This was the result of miss-communication.
The requested change was to rename `update_disk_health_cache` to something more descriptive.

Reply:
:)